src/commands/suggestions/suggestionsUpdate.ts

```diff
@@ -20,12 +20,19 @@ class SuggestionsUpdateCommand extends AdminCommand {
       memberName: 'update',
       args: [
         {
-          key: 'value',
-          prompt: `enter one of ${getAvailableStatesString()} before your IDs.`,
+          key: 'state',
+          prompt: `enter one of ${getAvailableStatesString()}.`,
           type: 'string',
           default: '',
-          validate: validateUpdate,
+          validate: validateState,
           parse: parseStateArg
+        },
+        {
+          key: 'ids',
+          prompt: `enter suggestion IDs seperated by spaces.`,
+          type: 'string',
+          default: '',
```

Comment:
Since we have defaults for these two arguments, what happens if none is provided? If you think these arguments are required, you don't need default values for them.

Reply:
I'm curious what would happen if you just do `suggestionupdate pending` without specifying IDs

Reply:
``suggestionupdate pending`` and ``suggestionupdate 1 2 3`` now give errors

Reply:
![image](https://user-images.githubusercontent.com/37851975/126252132-5fa55c11-c200-4c14-921c-5f0a3d34418d.png)

Reply:
> suggestionupdate pending and suggestionupdate 1 2 3 now give errors

Do they give error or do they prompt for user input?

Reply:
They error and give the cancel prompt (but at that point it'd be easier to just cancel)